azure-kusto-data/azure/kusto/data/aio/client.py

```diff
@@ -30,8 +30,13 @@ def __init__(self, kcsb: Union[KustoConnectionStringBuilder, str]):
     async def __aenter__(self) -> "KustoClient":
         return self
 
-    def __aexit__(self, exc_type, exc_val, exc_tb):
-        return self._session.__aexit__(exc_type, exc_val, exc_tb)
+    async def close(self):
+        if not self._is_closed:
+            await self._session.__aexit__(None, None, None)
+        super().close()
+
+    async def __aexit__(self, exc_type, exc_val, exc_tb):
```

Comment:
Hmmm
We can maybe give them `__enter__` and `__exit__`, but it's useless since they are not exposed to the user, and are only ever used as memebers (as opposed to local variables).
`__aenter__` doesn't make sense as they don't have an async initialization

Reply:
Lost you there. Is it something having to do with async python?
f2f?